okta_rules/okta_admin_role_assigned.py

```diff
@@ -20,7 +20,7 @@ def dedup(event):
                           'debugContext',
                           'debugData',
                           'requestId',
-                          default='REQUEST_ID_NOT_FOUND')
+                          default='UNKNOWN_REQUEST_ID')
```

Comment:
to conform to you other standard, should this be `<UNKNOWN_REQUEST_ID>` (add `<>` in these two?)